spec/unit/classes/master/config_spec.rb

```diff
@@ -1,99 +1,93 @@
 require 'spec_helper'
 
 describe 'puppetdb::master::config', :type => :class do
+  on_supported_os.each do |os, facts|
+    context "on #{os}" do
+      let(:facts) do
+        facts.merge({
+            :puppetversion => Puppet.version,
+            :fqdn => 'puppetdb.example.com',
```

Comment:
Apparently mocking node for some reason doesn't carry over to fqdn in these tests